organisations/organisations.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package organisations;
+
+option go_package = "grpc/organisationspb";
+
+import "google/protobuf/empty.proto";
+
+message Organisation {
+    string id = 1;
+
+    string name = 2;
+
+    Member creator = 3;
```

Comment:
In my opinion, `creator` sounds weird I would rename to `owner` or `admin`. Also I would do `string owner_id = 3;` or `string owner_account_id = 3;` because otherwise you're duplicating what is already stored inside of `members` right?